README.md

```diff
@@ -97,14 +97,13 @@ this suite:
 
 ### Selenium Tests
 
-this suite has not been implemented yet.
+run these with `pytest --base-url http://localhost:3000 --driver pytest-selenium/tests`
```

Comment:
Haha, yea you're right

Reply:
yea sorry.. I typically pass in Chrome as a default. Since we have a focus on cross browser it's something more explicit. I'm not used to it having to be passed in manually.